src/data_gradients/managers/abstract_manager.py

```diff
@@ -48,18 +44,14 @@ def __init__(
         :param train_data:          Iterable object contains images and labels of the training dataset
```

Comment:
please fix the docs: train_data and val_data are now BaseDatasetAdapter (not iterable)

Reply:
After the latest change, this will be `BaseSampleIterable` instead of `BaseDatasetAdapter`. `BaseSampleIterable` will be an iterable so we can leave the docstring like that